DOCS-1130: Test images classifications (#1981)
viamrobotics/docs: docs/services/vision/classification.md

## Test your classifier

You can test your classifier with [existing images in the Viam app](#existing-images-in-the-cloud), [live camera footage](#live-camera-footage), or [existing images on a computer](#existing-images-on-your-machine).

### Existing images in the cloud

{{< alert title="Note" color="note" >}}

> Your classifier must have been trained or uploaded after the September 19, 2023.

Reviewer: ```suggestion
The feature is only available for classifiers that were uploaded after September 19, 2023.
```

Unless the timestamp of training is also considered in the availability of this feature.

{{< /alert >}}

{{<gif webm_src="/services/vision/mug-classifier.webm" mp4_src="/services/vision/mug-classifier.mp4" alt="A classification model run against an image containing a mug." max-width="250px" class="alignright">}}

> If you have images stored in the [Viam cloud](/manage/data), you can run your classifier against your images in the app.

Reviewer: ```suggestion
If you have images stored in the [Viam cloud](/manage/data), you can run your classifier against your images in the [Viam app](https://app.viam.com/).
```

Let's link directly here; Step 1 links to a how-to, not to the target UI. So let's have the target UI link here.

> If you have images stored in the [Viam cloud](/manage/data), you can run your classifier against your images in the [Viam app](https://app.viam.com/).

Author: ```suggestion
If you have images stored in the [Viam cloud](/manage/data/), you can run your classifier against your images in the [Viam app](https://app.viam.com/).
```

1. Navigate to the [Data tab](/manage/data/view/) and click on the **Images** subtab.

> 2. Click on an image to open the side menu,select the **Actions** tab in the data tab.

Reviewer: ```suggestion
2. Click on an image to open the side menu, and select the **Actions** tab under the **Data** tab.
```
